dist/assembly.xml

```diff
@@ -161,6 +161,30 @@
       <useTransitiveFiltering>false</useTransitiveFiltering>
     </dependencySet>
 
+    <!-- Cassandra module -->
+    <dependencySet>
+      <outputDirectory>${hawkular.dist.zip.root.dir}/modules/org/apache/cassandra/main</outputDirectory>
+      <includes>
+        <include>org.apache.cassandra:cassandra-all:jar</include>
```

Comment:
By the way: I see no problem in changing the EAR to consume this Cassandra from the module, but I'd do it in phases. First, this change that should not affect other modules, then change the EAR, which should affect other modules.